test/ip.ts

```diff
@@ -0,0 +1,12 @@
+import test from 'ava';
+import withServer from './helpers/with-server';
+
+test('ip is defined', withServer, async (t, server, got) => {
```

Comment:
This should be IMO located in https://github.com/sindresorhus/got/blob/master/test/promise.ts

Reply:
The naming is a little irritating for me. It does not check that a promise is returned, but checks individual values of the result.

Reply:
> const {ip} = await got('');

You **do** check promises. You're using `await` to fetch the response.

Reply:
For me checking promises means to check if the promise itself is fullfilled or rejected. I'm checking if a attribute of the resolved value exists. But I got your point.